docs/UserGuide.md

```diff
@@ -734,14 +734,14 @@ If you would like to add a lesson to a class, you can make use of this command.
 * Adds a lesson at a specified `CLASS_INDEX`+ `DAY` + `START_TIME` + `END_TIME` + `VENUE` + `NO_OF_TIMES`.
 
 For example:
-* You receive news that you will be teaching `CS2103T Tutorial` on `Tuesday 10:00-11:00` at `COM2 #02-02` from week 1 to 13.
-Hence, you type in the command `add-lesson c\1 d\TUESDAY st\10:00 et\11:00 v\COM2 #02-02 r\13` and press <kbd>Enter</kbd>.
+1. You receive news that you will be teaching `CS2103T Tutorial` on `Tuesday 10:00-11:00` at `COM2 #02-02` from week 1 to 13.
+   Hence, you type in the command `add-lesson c\1 d\TUESDAY st\10:00 et\11:00 v\COM2 #02-02 r\13` and press <kbd>Enter</kbd>.
 
-  ![Add lesson before](images/ugimages/AddLessonBefore.png)
+   ![Add lesson before](images/ugimages/AddLessonBefore.png)
 
-* You have added a lesson to teach.
+1. You have added a lesson to teach.
 
-  ![Add lesson after](images/ugimages/AddLessonAfter.png)
+   ![Add lesson after](images/ugimages/AddLessonAfter.png)
```

Comment:
Perhaps it will be nice to annotate this diagram to show where the added lesson is?

Reply:
Agree with the annotations, but I think it is okay to dedicate a separate PR for that also.